### PR TITLE
8307369: Add execution of all svc tests in CI

### PR DESCRIPTION
**Pull request by lmesnik**
The fix excludes svc tests which should be updated to work with the virtual threads factory.
Also, ProcessTools updated to skip '-m' '--module' execution, which could not be run with test thread factory

---------
### Progress
- [x] Change must be properly reviewed (1 review required, with at least 1 [Reviewer](https://openjdk.org/bylaws#reviewer))
- [x] Change must not contain extraneous whitespace
- [x] Commit message must refer to an issue

### Issue
 * [JDK-8307369](https://bugs.openjdk.org/browse/JDK-8307369): Add execution of all svc tests in CI

### Reviewers
 * [Chris Plummer](https://openjdk.org/census#cjplummer) (@plummercj - **Reviewer**)
 * [Serguei Spitsyn](https://openjdk.org/census#sspitsyn) (@sspitsyn - **Reviewer**)

### Reviewing
<details><summary>Using <code>git</code></summary>

Checkout this PR locally: \
`$ git fetch https://git.openjdk.org/jdk.git pull/13968/head:pull/13968` \
`$ git checkout pull/13968`

Update a local copy of the PR: \
`$ git checkout pull/13968` \
`$ git pull https://git.openjdk.org/jdk.git pull/13968/head`

</details>
<details><summary>Using Skara CLI tools</summary>

Checkout this PR locally: \
`$ git pr checkout 13968`

View PR using the GUI difftool: \
`$ git pr show -t 13968`

</details>
<details><summary>Using diff file</summary>

Download this PR as a diff file: \
<a href="https://git.openjdk.org/jdk/pull/13968.diff">https://git.openjdk.org/jdk/pull/13968.diff</a>

</details>

### Webrev
[Link to Webrev Comment](https://git.openjdk.org/jdk/pull/13968#issuecomment-1546421997)

### CODE REVIEW

File: test/lib/jdk/test/lib/process/ProcessTools.java [hunk 1]
```diff
@@ -399,7 +399,7 @@ private static List<String> addMainWrapperArgs(String mainWrapper, List<String>
 
         final List<String> unsupportedArgs = List.of(
              "-jar", "-cp", "-classpath", "--class-path", "--describe-module", "-d",
-             "--dry-run", "--list-modules","--validate-modules", "-version");
+             "--dry-run", "--list-modules","--validate-modules", "-m", "--module", "-version");
```
Comment (plummercj) [inline test/lib/jdk/test/lib/process/ProcessTools.java:402]: Can you fix the indentation on line 398 above?

Comment (plummercj) [reply test/lib/jdk/test/lib/process/ProcessTools.java:402]: It doesn't seem to make much sense to have an argument be in both `unsupportedArgs` and `doubleWordArgs`. `unsupportedArgs` seems to take precedence and the arg will never have a change to be found in `doubleWordArgs`.

**Review by plummercj — CHANGES_REQUESTED** [review]

File: test/lib/jdk/test/lib/process/ProcessTools.java (hunk 1 above)
Comment (lmesnik) [reply test/lib/jdk/test/lib/process/ProcessTools.java]: fixed

**Review by plummercj — APPROVED** [review]
Changes look good. Can you make sure all the CRs in question have the problemlist label added and a link back to this CR?

**Pull request closed without merging**